traitsui/testing/tester/base_classes.py

```diff
@@ -0,0 +1,44 @@
+#  Copyright (c) 2005-2020, Enthought, Inc.
```

Comment:
Naming things is hard... This module should be renamed, I just wanted to make it something for now.

Reply:
Agreed this needs to be renamed. Let's see what we ended up with and then we can name it more appropriately before we merge this.